src/schema/home/results.js

```diff
@@ -108,6 +108,33 @@ const moduleResults = {
       sort: "-position",
     })
   },
+  similar_to_saved_works: ({
+    rootValue: { savedArtworksLoader, similarArtworksLoader, artworksLoader },
+  }) => {
+    return savedArtworksLoader({
+      size: RESULTS_SIZE,
+      sort: "-position",
+    }).then(works => {
+      return similarArtworksLoader(
+        { artwork_id: map(works, "_id").slice(0, 7) }
+      ).then(ids => {
+        return artworksLoader({ ids })
```

Comment:
Confused about this...doesn't `similarArtworksLoader` already return the artworks? I think the `.then(ids => artworksLoader` isn't needed?

Reply:
It only returns an array of IDs right now

Reply:
i could update the API after confirming it's not in use by clients

Reply:
I thought `/api/v1/related/artworks` already returns artworks and not ids? I'm very confused...

https://github.com/artsy/gravity/blob/f6da1d869c406708696781b5ba07dc1421a90c41/app/api/v1/related_endpoint.rb#L270

Reply:
Ah I see, https://api.artsy.net/api/v1/related/artworks?artwork_id[]=andy-warhol-skull&artwork_id[]=juan-gris-woman does indeed return an array of ids.

That endpoint in gravity looks odd to me:
  - why reuse `artwork_id` in the plural way, vs adding `artwork_ids`?
  - why is there a `.to_json(properties: :...)` if its not materializing JSON?

Lol so weird

In that case 👍 carry on!

Reply:
Cool. Wondering if it makes sense for that endpoint to just return actual artworks rather than ids? That could obviously be a future thing (and might break existing consumers). It sort of seems like while that endpoint is already doing quite a lot, looking up the actual artworks vs returning the id's might be reasonable.

Reply:
Sweet, we can drop these now!